website/docs/release_notes.md

```diff
@@ -5,6 +5,140 @@ sidebar_position: 99
 
 # Release notes
 
+## [v0.1.15](https://github.com/VirtusLab/scala-cli/releases/tag/v0.1.15)
+
+## M1 native launcher is here! (experimental)
+
+We are happy to announce that there is new dedicated launcher for M1 users.
+
+Added in [#1396](https://github.com/VirtusLab/scala-cli/pull/1396) by [@lwronski](https://github.com/lwronski)
+
+## `--python` option for `repl` sub-command (experimental)
+
+Passing `--python` option allows using ScalaPy with `repl`:
+
+```
+▶ scala-cli --python
+Welcome to Scala 3.1.3 (17.0.2, Java OpenJDK 64-Bit Server VM).
+Type in expressions for evaluation. Or try :help.
+                                                                                
+scala> import me.shadaj.scalapy.py
+                                                                                
+scala> py.Dynamic.global.range(1, 4)
+val res0: me.shadaj.scalapy.py.Dynamic = range(1, 4)
+```
+
+Added in [#1336](https://github.com/VirtusLab/scala-cli/pull/1336) by [@alexarchambault](https://github.com/alexarchambault)
+
+## `-d`, `-classpath` and `compile` sub-command's `--output` options changes
+
+To be backward compatible with `scala` command a lot of changes have been made to the following options:
+* The `compile`'s  `--output` option has been renamed to `--compilation-output`. This option is now also available from the `run` and `package` sub-commands:
```

Comment:
```suggestion
* The `compile` sub-command's  `--output` option has been renamed to `--compilation-output`. This option is now also available from the `run` and `package` sub-commands.
```
can't end with colon here, as that'd somewhat imply the example coming next would have to include using it with the `run` and `package` sub-commands (which I don't think is really necessary)